examples/bpk-component-map/examples.js

```diff
@@ -276,6 +275,82 @@ class StatefulBpkIconMarker extends Component<
   }
 }
 
+const getPixelPositionOffset = (width: number, height: number) => ({
+  x: -(width / 2),
+  y: -height,
+});
+
+const StatefulBpkPriceMarkerButtonWithPopoverOnMap = ({
+  action: inlineAction,
+  airportsIconWithPrice,
+}: {
+  action: () => mixed,
+  airportsIconWithPrice: boolean,
+}) => {
+  const [selectedId, setSelectedId] = useState('2');
+  const [viewedVenues, setViewedVenues] = useState(['1']);
+  const ref = useRef(null);
+
+  const getStatus = (id: string) => {
+    if (selectedId === id) {
+      return MARKER_STATUSES.selected;
+    }
+    if (viewedVenues.includes(id)) {
+      return MARKER_STATUSES.previous_selected;
+    }
+    return MARKER_STATUSES.unselected;
+  };
+
+  const selectVenue = (id: string) => {
+    setSelectedId(id);
+    setViewedVenues((prevState) => [...prevState, id]);
+  };
+
+  return (
+    <StoryMap zoom={15} center={{ latitude: 55.944665, longitude: -3.1964903 }}>
+      {venues.map((venue) => (
+        <BpkOverlayView
+          getPixelPositionOffset={getPixelPositionOffset}
+          position={{
+            latitude: venue.latitude,
+            longitude: venue.longitude,
+          }}
+        >
+          <BpkPopover
+            hoverable
+            showArrow
+            id="my-popover}"
+            padded
+            label="My popover"
+            labelAsTitle
+            target={
+              <div ref={ref} style={{ width: 'fit-content' }}>
+                <BpkPriceMarkerButton
+                  id={venue.id}
+                  label={venue.price}
+                  icon={airportsIconWithPrice ? venue.airportsIcon : null}
+                  position={{
+                    latitude: venue.latitude,
+                    longitude: venue.longitude,
```

Comment:
it seems no `position` prop in `BpkPriceMarkerButton`

Reply:
Moved.